### PR TITLE
AAP-5297 Added attributes and corrected accessibility options

### PR DESCRIPTION
**Pull request by ianf77**
Add attributes and correct accessibility options

https://issues.redhat.com/browse/AAP-5297

Affects `titles/hub/configuring-private-hub-rh-certified`: 'Managing Red Hat Certified and Ansible Galaxy collections in Automation Hub'

### CODE REVIEW

**Comment by ianf77** [comment]
This is  a replacement for PR#480, as subsequent pushes to that PR kept creating new PRs.

File: downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc [hunk 1]
```diff
@@ -3,23 +3,25 @@
...
-. Navigate to *Automation Hub* -> *Repo Management*.
-. Locate the *Sync URL* and click image:images/copy.png[Copy to clipboard,10,25]. Paste the *Sync URL* in a file to use when configuring the rh-certified remote.
-. Click image:images/more_actions.png[More,10,25] and click *Get token*.
-. On the *Token management* page, click *Load token*.
-. Click *Copy to clipboard* to copy the API token.
+. Log in to console.redhat.com as an Organization Administrator.
+. Navigate to menu:Automation Hub[Repo Management].
+. Locate the *Sync URL* and click image:images/copy.png[Copy to clipboard,10,25]. 
+Paste the *Sync URL* in a file to use when configuring the rh-certified remote.
+. Click image:images/more_actions.png[More,10,25] and click btn:[Get token].
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc:18]: The more actions icon doesn't look very clear in my docs build. Would it make sense to say something like
```suggestion
. Click the btn:[more actions] icon (image:images/more_actions.png[More,10,25]) and click btn:[Get token].
```

File: downstream/modules/automation-hub/proc-set-community-remote.adoc [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 See https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_user_access_in_private_automation_hub/index[Managing user access in Automation Hub] for more information on permissions.
-* You have a `requirements.yml` file that identifies those collections to sync from Ansible Galaxy.
+* You have a `requirements.yml` file that identifies those collections to sync from {Galaxy}.
 See example below.
 
 .Requirements.yml example
 -----
 collections:
-  # Install a collection from Ansible Galaxy.
+  # Install a collection from {Galaxy}.
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-set-community-remote.adoc:19]: The `{Galaxy}` attribute doesn't work within a codeblock.
```suggestion
# Install a collection from Ansible Galaxy.
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
@ianf77 
Need to remove the {Galaxy} attribute from a codeblock; my other comment is just an opinionated suggestion that you can ignore if you like.
Otherwise, LGTM

File: downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc (hunk 1 above)
Comment (ianf77) [reply downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc:18]: The icons are pretty awful in a bccutil build too, but look OK in the published doc.

File: downstream/modules/automation-hub/proc-set-community-remote.adoc (hunk 2 above)
Comment (ianf77) [reply downstream/modules/automation-hub/proc-set-community-remote.adoc:19]: attributes don't seem to work in the master.adoc title either

Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-set-community-remote.adoc]: They look OK after a rebase against `main`

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**